src/flp/types/fixedpoint_l2.rs

```diff
@@ -576,16 +652,32 @@ where
     }
 }
 
+impl<T, SPoly, SBlindPoly> TypeWithNoise<ZCdpDiscreteGaussian>
+    for FixedPointBoundedL2VecSum<T, SPoly, SBlindPoly>
+where
+    T: Fixed + CompatibleFloat,
+    SPoly: ParallelSumGadget<Field128, PolyEval<Field128>> + Eq + Clone + 'static,
+    SBlindPoly: ParallelSumGadget<Field128, BlindPolyEval<Field128>> + Eq + Clone + 'static,
+{
+    fn add_noise_to_result(
+        &self,
+        dp_strategy: &ZCdpDiscreteGaussian,
+        agg_result: &mut [Self::Field],
+        _num_measurements: usize,
+    ) -> Result<(), FlpError> {
+        self.add_noise(dp_strategy, agg_result, &mut SeedStreamSha3::from_entropy())
```

Comment:
How about in-lining the implementation of add_noise?

Reply:
See [here](https://github.com/divviup/libprio-rs/pull/578#discussion_r1268485011).